cbr-service-profile/outputs.tf

```diff
@@ -0,0 +1,33 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "zone_id" {
```

Comment:
names and descriptions of the outputs should be updated to state multiples are returned eg, zone_ids, zone_crns etc